src/main/java/nextstep/auth/application/AuthService.java

```diff
@@ -28,4 +30,19 @@ public TokenResponse login(TokenRequest tokenRequest) {
         String token = jwtTokenProvider.createToken(member.getEmail(), member.getRoles());
         return new TokenResponse(token);
     }
+
+    public TokenResponse loginGithub(String code) {
```

Comment:
member 생성이 안되고 있을 것 같아요 😄 
```suggestion
   @Transactional
    public TokenResponse loginGithub(String code) {
```

Reply:
놓친 부분이 많군요 .. ㅠㅠ 감사합니다!